wallet/bindings/python/tox.ini

```diff
@@ -14,5 +14,5 @@ description = Run the unit tests under {basepython}
 passenv = *
 deps = -r requirements-dev.txt
 commands =
-    pip install .
+    python setup.py install
```

Comment:
Also undo this change if you do the other changes

Reply:
Yes sir